torch/_torch_docs.py

```diff
@@ -662,6 +714,35 @@ def merge_dicts(*dicts):
     tensor([ 0.9833,  0.0811, -1.9743, -1.4151])
 """.format(**common_args))
 
+add_docstr(torch.atanh,
+           r"""
+atanh(input, out=None) -> Tensor
+
+Returns a new tensor with the inverse hyperbolic tangent of the elements of :attr:`input`.
+The domain of the inverse hyperbolic tangent is `(-1, 1)` and values outside this range
+will be mapped to ``NaN``, except for the values `1` and `-1` for which the output is
+mapped to `+/-INF` respectively.
+
+.. math::
+    \text{out}_{i} = \tanh^{-1}(\text{input}_{i})
+""" + r"""
+Args:
+    input (Tensor): the input tensor where math:`input` is in the open interval `(-1, 1)`
```

Comment:
Can just remove this, like add, since all the info will be in the note.

Reply:
Done! Thanks, Ruberry.